Cargo.toml

```diff
@@ -46,10 +46,23 @@ thiserror = "1.0.29"
 url = "2.2.2"
 webbrowser = { version = "0.8.0", optional = true }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+getrandom = { version = "0.2.3", features = ["js"] }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+getrandom = "0.2.3"
+
 [dev-dependencies]
 env_logger = { version = "0.11.0", default-features = false }
-tokio = { version = "1.11.0", features = ["rt-multi-thread", "macros"] }
 futures-util = "0.3.17"
+wasm-bindgen-test = "0.3.34"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
+tokio = { version = "1.11.0", features = ["rt-multi-thread", "macros"] }
+
+[target.'cfg(target_arch = "wasm32")'.dev-dependencies]
+tokio = { version = "1.11.0", features = ["rt", "macros"] }
```

Comment:
Strictly speaking tokio isn't needed for running wasm tests, but to remove it would have made the `#[cfg...` attributes bigger